axon-server-connector/src/main/java/org/axonframework/axonserver/connector/util/UpstreamAwareStreamObserver.java

```diff
@@ -27,4 +27,18 @@ public void beforeStart(ClientCallStreamObserver<Object> requestStream) {
     public ClientCallStreamObserver<?> getRequestStream() {
         return requestStream;
     }
+
+    /**
+     * Completes the request steam related to this stream observer. Ignores exceptions that may occur (for instance if
+     * the request stream is already half-closed).
+     */
+    protected void completeRequestStream() {
+         if( requestStream != null) {
```

Comment:
There is a StreamObserverUtils that contains this logic already. Can't we use that one instead?

Reply:
StreamObserverUtils is class in axonserver se

Reply:
🤦‍♂